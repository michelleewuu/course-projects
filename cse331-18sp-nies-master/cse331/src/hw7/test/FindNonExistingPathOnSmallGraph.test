# Tests find path on smallGraph.

# Load smallGraph.tsv
LoadGraph smallGraph smallGraph.tsv

# Find minimum-cost path from NodeA to NodeC
FindPath smallGraph NodeA NodeC
