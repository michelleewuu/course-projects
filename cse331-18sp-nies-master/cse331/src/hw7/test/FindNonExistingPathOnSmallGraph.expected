# Tests find path on smallGraph.

# Load smallGraph.tsv
loaded graph smallGraph

# Find minimum-cost path from NodeA to NodeC
path from NodeA to NodeC:
no path found
